### PR TITLE
RD-5107: Indicate installation failure on GettingStartedModal

### PR DESCRIPTION
**Pull request by NorbertAmartus**
## Description

Changes consists of:
- Displaying information about the installation failure (with a dedicated label and adequate progress bar color)
- Sliming `SummaryStep` component by extracting a new `TaskList` component 🚀 
- Introducing new test scenario to `getting_started_spec.tsx`

### Before
Before

### After 
After

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/3041/)

## Documentation

N/A

### CODE REVIEW

File: app/translations/en.json [hunk 1]
```diff
@@ -120,7 +120,8 @@
             "blueprintUploadError": "{{blueprintName}} blueprint upload error:\n{{uploadError}}",
             "timeoutExceededError": "Timeout exceeded.",
             "progressMessage": "Installation in progress...",
-            "doneMessage": "Installation completed! (you can now close this window)"
+            "doneMessage": "Installation completed! (you can now close this window)",
+            "failureMessage": "Installation failed."
```
Comment (qooban) [inline app/translations/en.json:124]: Optional:

Probably we can also mention that the reason for failure (errors description) is presented at the top of the modal:
image

**Review by qooban — APPROVED** [review]

File: app/translations/en.json (hunk 1 above)
Comment (NorbertAmartus) [reply app/translations/en.json:124]: I was wondering about it too (I have even [left a comment](https://cloudifysource.atlassian.net/browse/RD-5107?focusedCommentId=76625) with that suggestion), but the [PM said](https://cloudifysource.atlassian.net/browse/RD-5107?focusedCommentId=76626) that the message should be just "Installation failed".

Comment (qooban) [reply app/translations/en.json:124]: Right, seems like we've done all we could do 🙂
I'm resolving this discussion.

**Review by PatrycjaKubiczek — APPROVED** [review]
LGTM! 🚀 
it makes much more sense to get "Installation failure" status while we're getting errors 👍

**Pull request merged**